### PR TITLE
fix: update type error

### PR DESCRIPTION
**Pull request by vunguyendhSTS**

### CODE REVIEW

File: template/src/store/reducers/app.ts [hunk 1]
```diff
@@ -8,24 +7,26 @@ const initialState: IApp = {
...
+      return state.appState
+    },
+    [APP_CONSTANTS_ACTIONS.GET_SETTING_APP_ACTIONS.SUCCESS]: () => {},
+    [APP_CONSTANTS_ACTIONS.GET_SETTING_APP_ACTIONS.FAILURE]: () => {},
+    [APP_CONSTANTS_ACTIONS.LOGIN_ACTIONS.HANDLER]: () => {},
+    [APP_CONSTANTS_ACTIONS.LOGIN_ACTIONS.SUCCESS]: () => {},
+    [APP_CONSTANTS_ACTIONS.LOGIN_ACTIONS.FAILURE]: () => {},
+    getSettings: () => {},
+    setAppStack: (state, action: PayloadAction<string>) => {
       state: state.appState = action.payload
```
Comment (ngochuyduong) [inline template/src/store/reducers/app.ts:27]: Reduce unnecessary state prop with ES6 syntax

File: template/src/store/reducers/app.ts [hunk 2]
```diff
@@ -8,24 +7,26 @@ const initialState: IApp = {
...
     },
-    getSettingsSuccess: (state: IApp, action: IAction): void => {},
-    setShowGlobalIndicator: (state: IApp, action: IAction): void => {
+    setShowGlobalIndicator: (state, action: PayloadAction<boolean>) => {
       state: state.showGlobalIndicator = action.payload
```
Comment (ngochuyduong) [inline template/src/store/reducers/app.ts:30]: Reduce unnecessary state prop with ES6 syntax

**Review by ngochuyduong — CHANGES_REQUESTED** [review]

**Comment by vunguyendhSTS** [comment]
This PR combine on #10 PR so i close this PR

**Comment by vunguyendhSTS** [comment]
This PR combine on #10 PR so i close this PR

**Pull request closed without merging**